### PR TITLE
CDC

### PR DESCRIPTION
**Pull request by gitr0n1n**
+ CDC 15

### CODE REVIEW

**Review by TheEnthusiasticAs — APPROVED** [review]

**Review by TheEnthusiasticAs — APPROVED** [review]

File: content/blog/2020-09-25-core-devs-call-2020-ecip-1092-1097-last-call/index.en.md [hunk 1]
```diff
@@ -10,13 +10,18 @@ tags: ["announcement", "development", "education", "teams", "hardfork", "media"]
...
 * When: Friday, September 25, 2020, 4pm UTC, 90 minutes max.
-* Where: Jitsi link will be published ahead of the call.
-* Focus: 51% Attack Solutions
+* Where: [Discord](https://discord.gg/hQs894U)
+* Focus: Mining Algorithm, Last-Call Proposals, and other 51%-Attack Solutions.
 
 ## Agenda
 
+51% Proposals
 * ECIP 1092 PIRLGUARD (Draft)
```
Comment (developerkevin) [inline content/blog/2020-09-25-core-devs-call-2020-ecip-1092-1097-last-call/index.en.md]: Could we remove this line regarding pi

File: content/blog/2020-09-25-core-devs-call-2020-ecip-1092-1097-last-call/index.en.md [hunk 2]
```diff
@@ -10,13 +10,18 @@ tags: ["announcement", "development", "education", "teams", "hardfork", "media"]
...
-* ECIP 1097 Ouroboros BFT Checkpointing (Draft)
```
Comment (developerkevin) [inline content/blog/2020-09-25-core-devs-call-2020-ecip-1092-1097-last-call/index.en.md:19]: Recommending to leave this line in. IOHK devs / researchers were on the call to discuss and answer questions by participants.

Would also suggest adding "ECIP-1096 ECIP 1096: 51% Attack protection system based on Bitcoin Merged Mining"  as they were also in the call explaining and answering questions about their proposal. https://github.com/ethereumclassic/ECIPs/pull/344

These two along with MESS were the only 51% solutions discussed by respective authors.

**Review by developerkevin — CHANGES_REQUESTED** [review]
LGTM. Except for the comments I've made.

File: content/blog/2020-09-25-core-devs-call-2020-ecip-1092-1097-last-call/index.en.md (hunk 1 above)
Comment (gitr0n1n) [reply content/blog/2020-09-25-core-devs-call-2020-ecip-1092-1097-last-call/index.en.md]: Dated Comment. Resolved.

File: content/blog/2020-09-25-core-devs-call-2020-ecip-1092-1097-last-call/index.en.md (hunk 2 above)
Comment (gitr0n1n) [reply content/blog/2020-09-25-core-devs-call-2020-ecip-1092-1097-last-call/index.en.md:19]: Resolved. These were added. I believe you're viewing a dated version of this file. My last commit should have added all the notes posted after the meeting.

**Review by developerkevin — APPROVED** [review]
LGTM

**Comment by gitr0n1n** [comment]
Can one of the reviewers please approve. This has a lot of the video presentations and CDC calls in it. So a high information PR for people to follow all the evolving convo. Thanks much!

**Review by hitchcott — APPROVED** [review]

**Pull request merged**